content/post/contributors/yokwejuste.md

```diff
@@ -0,0 +1,12 @@
+#Hey I'm yokwejuste 😁
+***Best slogan:*** : Beauty better than ugly.
```

Comment:
Minor nitpick, if you want to say "beauty", you should probably use "ugliness" and if you want to use "ugly" you should probably say "beautiful" otherwise it's a bit dissonant imho.

Reply:
Thanks, for that.

Reply:
**Best Slogan** : Beauty better than ugliness, clean code is the objective.